extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/app.js

```diff
@@ -105,11 +105,15 @@ angular.module('PaperUI', [ 'PaperUI.controllers', 'PaperUI.controllers.control'
             redirectTo : '/control'
         });
     }
+    $mdDateLocaleProvider.shortMonths = [ 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec' ];
+    if (window.localStorage.getItem('paperui.language') == 'de') {
+        $mdDateLocaleProvider.shortMonths = [ 'Jan', 'Feb', 'Mär', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Dez' ];
+    }
     $mdDateLocaleProvider.formatDate = function(date) {
         if (!date) {
             return null;
         }
-        return (date.getFullYear() + '-' + (date.getMonth() + 1) + '-' + date.getDate());
+        return (date.getDate() + '.' + (date.getMonth() + 1) + '.' + date.getFullYear());
```

Comment:
We discussed it a little bit. The format with 'day first' is not necessarily English as British format also uses day first. The format with 'month first' is American and I think it should not be localized based on language. If we need to localize it we should probably use European and American date format option in Preferences. What do you think?